weblab_cg/_generator_chaste_cg.py

```diff
@@ -537,10 +537,9 @@ def _format_derivative_equations(self):
                 divisor_unit = self._get_desired_units(d_eqs[i].lhs.args[1][0])
                 units = dividend_unit / divisor_unit
                 units_from = self._model.units.summarise_units(d_eqs[i].lhs)
-                if units.dimensionality == units_from.dimensionality:
-                    factor = self._model.units.get_conversion_factor(units, from_unit=units_from)
-                else:
-                    factor = 1.0
+                factor = self._model.units.get_conversion_factor(units, from_unit=units_from) \
```

Comment:
That's a bit of a cheat way of fixing coverage ;)

Reply:
I Know, however I know the else case will be covered for the next model I'm trying. i just decided when adding that model I should really split up PRs, and I'd rather not take out the check and then again spent time on wondering why the next model isn't working. So the next model will use the else case, promise!